docs/en/latest/how-to-build.md

```diff
@@ -54,6 +54,9 @@ sudo yum --showduplicates list apisix
 
 # Will install the latest apisix package
 sudo yum install apisix
+
+# Will install the specified version 2.10.3 apisix package
+sudo yum install apisix-2.10.3-0.el7
```

Comment:
and please remove this,thx

Reply:
We encourage developers to install and use the latest Apache APISIX version, @moonming you'd better leave your reason why remove this :) Because not all contributors/users know this.

Reply:
This modification has nothing to do with the title

Reply:
Hi @bin-ya, usually one PR only does one thing 😄 This is helpful for reviewers and community project management.